lib/ansible/modules/cloud/amazon/rds.py

```diff
@@ -586,7 +586,8 @@ def get_data(self):
             d["endpoint"] = None
             d["port"] = None
             d["vpc_security_groups"] = None
-
+        if self.instance.DBName is not None:
```

Comment:
This could just be `if self.instance.DBName:` I think since the database name is either None or made up of alpha-numeric characters, starting with a letter.